index.bs

```diff
@@ -14,4 +14,21 @@ Group: WebView-CG
 
 # Introduction # {#introduction}
 
-Hello world!
\ No newline at end of file
+This specification describes a content embedding API that satisfies some
+critical use cases that <{iframe}> does not support. This embedding environment
+should allow embedding all content, including content which <{iframe}> cannot
+embed, and provide embedding sites more controlover that embedded content.
+Additionally, embedding a site's content should be possible without express
+permission from the embedded site.
```

Comment:
Re: "should allow embedding all content" and "should be possible without express permission", I think these 2 phrases refer to the same capability (ignoring X-Frame-Options). Do they? If so, let's just refer to it once here by merging the statements together.

Reply:
Yes, they do go over the same capability. I merged these two as you suggested. Thanks!